tests/unit/Map.ts

```diff
@@ -57,7 +56,7 @@ registerSuite({
 	},
 
 	'delete': {
-		before() {
+		before(this: any) {
```

Comment:
You shouldn't need to provide a type for `this` in a test (and this method doesn't even appear to use `this`).